diff --git a/core.js b/core.js
index a5e82c2289b0523855a0f9f5023de438bbfbeef2..a5eb90db57829a73f8efe947cbd7da8dcb3c6889 100644
--- a/core.js
+++ b/core.js
@@ -251,7 +251,7 @@ class Resource {
 }
 _Resource_rid = new WeakMap();
 function isTauri() {
-    return 'isTauri' in window && !!window.isTauri;
+    return 'isTauri' in (globalThis || window) && !!(globalThis || window).isTauri;
 }
 
 export { Channel, PluginListener, Resource, SERIALIZE_TO_IPC_FN, addPluginListener, checkPermissions, convertFileSrc, invoke, isTauri, requestPermissions, transformCallback };
